### PR TITLE
Claim multiple prizes

### PR DESCRIPTION
**Pull request by asselstine**

### CODE REVIEW

File: src/Claimer.sol [hunk 1]
```diff
@@ -56,47 +47,43 @@ contract Claimer is Multicall {
     }
 
     /// @notice Allows the call to claim prizes on behalf of others.
-    /// @param drawId The draw id to claim prizes for
-    /// @param _claims The prize claims
```
Comment (PierrickGT) [inline src/Claimer.sol:60]: Replace by the Natspec doc for the new params.

File: src/Claimer.sol [hunk 2]
```diff
@@ -56,47 +47,43 @@ contract Claimer is Multicall {
...
-            perTimeUnit = LinearVRGDALib.getPerTimeUnit(prizePool.estimatedPrizeCount(), drawPeriodSeconds);
-            elapsed = block.timestamp - (prizePool.lastCompletedDrawStartedAt() + drawPeriodSeconds);
+    ) external returns (uint256 totalFees) {
+
+        uint256 claimCount;
+        for (uint i = 0; i < winners.length; i++) {
+            claimCount += prizeIndices[i].length;
         }
 
         // compute the maximum fee based on the smallest prize size.
```
Comment (PierrickGT) [inline src/Claimer.sol:65]: Is this comment still relevant?

File: src/Claimer.sol [hunk 3]
```diff
@@ -105,7 +92,7 @@ contract Claimer is Multicall {
         for (uint i = 0; i < _claimCount; i++) {
             fee += _computeFeeForNextClaim(minimumFee, decayConstant, perTimeUnit, elapsed, prizePool.claimCount() + i, maxFee);
         }
-        return fee;
+        return (fee / _claimCount) * _claimCount; // round it out to match claimPrizes()
```
Comment (PierrickGT) [inline src/Claimer.sol:95]: We lose a bit in precision here by dividing and then multiplying.
Why not just create a function `_computeFeePerClaim` that we could use in `claimPrizes`?

**Review by PierrickGT — CHANGES_REQUESTED** [review]

File: src/Claimer.sol (hunk 3 above)
Comment (asselstine) [reply src/Claimer.sol:95]: It's supposed to lose precision!  That being said that's a good refactoring

**Pull request merged**